src/www/ui/ui-export-list.php

```diff
@@ -386,13 +386,17 @@ public function getCopyrights($uploadId, $uploadtree_pk, $uploadTreeTableName,
     $agentName = "copyright";
```

Comment:
Please update `$agentName` to an array as done in `src/spdx2/agent/spdx2.php` above.